include/ignition/rendering/LightVisual.hh

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef IGNITION_RENDERING_LIGHTVISUAL_HH_
+#define IGNITION_RENDERING_LIGHTVISUAL_HH_
+
+#include "ignition/rendering/config.hh"
+#include "ignition/rendering/Object.hh"
+#include "ignition/rendering/RenderTypes.hh"
+#include "ignition/rendering/Visual.hh"
+
+namespace ignition
+{
+  namespace rendering
+  {
+    inline namespace IGNITION_RENDERING_VERSION_NAMESPACE {
+
+      /// \brief Enum for LightVisual types
+      enum IGNITION_RENDERING_VISIBLE LightVisualType
+      {
+        LightVisual_NONE                = 0,
```

Comment:
add `/// \brief`

Reply:
can you also change the naming convention of these enums to match others in ign-rendering,
 e.g.
https://github.com/ignitionrobotics/ign-rendering/blob/ign-rendering4/include/ignition/rendering/Marker.hh#L38

 so this would be something like `LVT_NONE`

Reply:
There is a conflict with LidarVisualType. Any recomendations ?

Reply:
ah that's unfortunate, how about `LVT_EMPTY`?

Reply:
@iche033 Resolved! https://github.com/ignitionrobotics/ign-rendering/pull/202/commits/69232c03650aa1784a35b6b75a134706610b11ad